examples/generate_suit_encryption_info_aesgcm.py

```diff
@@ -94,8 +94,8 @@
 
 # The sender side:
 r = Recipient.new(
-    protected={},
-    unprotected={"alg": "ECDH-ES+A128KW"},
+    protected={"alg": "ECDH-ES+A128KW"},
```

Comment:
Bug fix: As the ECDH-ES+AES-KW CDDL says below, moved alg into inner protected header.
```
recipient_header_map_esdh = {
    1 => int,         ; algorithm identifier
  * label => values   ; extension point
}
```